lib/ruboty/github_pr_release/actions/release.rb

```diff
@@ -29,6 +29,8 @@ def create
           message.reply("Could not find that repository")
         rescue Octokit::UnprocessableEntity
           message.reply("No commits between #{from_branch} and #{base}")
+        rescue PrExistsError
+          message.reply("The Pull Request has already opened")
```

Comment:
Can you change the wording to the following?

 "This pull request is already open"